netuitive/client.py

```diff
@@ -112,11 +116,19 @@ def post(self, element):
                 logging.exception('Posting has been disabled.'
                                   'See previous errors for details.')
             else:
+                self.post_error_count += 1
+                if self.post_error_count > self.max_post_errors:
+                    element.clear_samples()
```

Comment:
correct.  Little, if any, point in keeping the data if we are still in a error state.